### PR TITLE
Convert Sched2Test to Kotlin

### PR DESCRIPTION
**Pull request by dae**
Useful for the backend work I'm working on

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/libanki/sched/SchedV2Test.kt [hunk 1]
```diff
@@ -197,213 +90,255 @@ public class SchedV2Test extends RobolectricTest {
...
-        note.setItem("Back", "two");
-        col.addNote(note);
-        col.reset();
-        assertEquals(1, col.getSched().newCount());
+        val note = col.newNote()
+        note.setItem("Front", "one")
+        note.setItem("Back", "two")
+        col.addNote(note)
+        col.reset()
+        Assert.assertEquals(1, col.sched.newCount().toLong())
```
Comment (mikehardy) [inline AnkiDroid/src/test/java/com/ichi2/libanki/sched/SchedV2Test.kt:330]: @david-allison I seem to recall other KotlinCleanups around toLong? is there one needed for this class?

**Review by mikehardy — APPROVED** [review]
seems fine w/two cleanup annotation questions - one definite, one I asked David on. Happy to see this go in quickly to keep from blocking progress, cleanups are minor points

**Review by lukstbit — APPROVED** [review]
Looks good, with one issue. Add a kotlin cleanup on the SchedV2Test class:

`@KotlinCleanup("fix ide lint issue and improve kotlin code where possible")`

**Comment by dae** [comment]
Done

**Pull request merged**